grpc_check/manifest.json

```
@@ -0,0 +1,38 @@
+{
+  "display_name": "gRPC Check",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "grpc_check",
+  "metric_prefix": "grpc_check.",
+  "metric_to_check": [
+    "grpc_check.healthy",
+    "grpc_check.unhealthy"
+  ],
+  "creates_events": false,
+  "short_description": "Monitor gRPC servers based on gRPC Health Checking Protocol",
+  "guid": "c315801f-de1b-4c4c-974f-da2e76f245db",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "grpc_check",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": true,
+  "integration_id": "grpc-check",
```

Hey @keisku, looks like the integration tile logo request was created with the `integration_id: grpc_check`. Do you mind changing this to match `grpc_check` or would you prefer that the designOps team update the logos to use the integration_id `grpc-check`? 
```suggestion
  "integration_id": "grpc_check",
```

---

>  would you prefer that the designOps team update the logos to use the integration_id grpc-check?
Yes!

integration_id should be `grpc-check`.

```
Executing validation manifest
Validating manifest.json files for 1 checks ...
grpc_check/manifest.json... FAILED
  integration_id Error: 'grpc_check' does not match '^[a-z][a-z0-9-]{0,254}(?<!-)$'
  skipping check for changed fields: integration not on default branch
1 invalid files
```